rcl/src/rcl/security_directory.c

```diff
@@ -112,12 +104,22 @@ static bool get_best_matching_directory(
         memcpy(matched_name, file.name, max_match_length);
       }
     }
-    if (-1 == tinydir_next(&dir)) {
+    rcutils_ret_t ret = rcutils_file_fini(&file, dir.allocator);
+    if (ret != RCUTILS_RET_OK) {
+      RCL_SET_ERROR_MSG_WITH_FORMAT_STRING("%s", rcutils_get_error_string().str);
+      rcutils_reset_error();
```

Comment:
Aren't the `rcl` error msg functions just a rename of the `rcutils` error msg functions? Isn't the reset effectively resetting the error message set in the line before?

Reply:
removed rcutils_reset_error() https://github.com/ros2/rcl/pull/605/commits/f6730a02341c9220677c99e005f314b718f9c947